gui/CMakeLists.txt

```diff
@@ -0,0 +1,66 @@
+cmake_minimum_required(VERSION 2.8)
+project(gui)
+cmake_policy(SET CMP0015 NEW)
+# Find includes in corresponding build directories
+set(CMAKE_INCLUDE_CURRENT_DIR ON)
+# Instruct CMake to run uic automatically when needed
+set(CMAKE_AUTOUIC ON)
+# Instruct CMake to run moc automatically when needed
+set(CMAKE_AUTOMOC ON)
+set(CMAKE_PREFIX_PATH $ENV{HOME}/Qt5.7.0/5.7/gcc_64)
```

Comment:
QT may not be found there.  This sounds very environment specific.  CMake should have a way to check that in a generic fashion.